### PR TITLE
Update readme with CLI and directory structure details for end user

### PR DESCRIPTION
**Pull request by shelbyspees**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,108 @@
```
Comment (smaslennikov) [inline README.md]: Please add a TOC: https://github.com/grindrllc/isre-tools-dotfiles/blob/master/bin/gh-md-toc

Comment (smaslennikov) [reply README.md]: This first line is empty and useless?

File: README.md [hunk 2]
```diff
@@ -2,9 +2,108 @@
...
 
-## Building
+## Using Revolution
 
-To setup a build environment, first boot the Vagrant VM:
+### Installing the gem
+
+Add the Revolution gem to your **Gemfile** to [have Bundler install it from GitHub](http://bundler.io/v1.16/guides/git.html).
+
+```ruby
```
Comment (joshuajlai) [inline README.md]: Can you put a comment line here to say the file name this logic goes into? I noticed other ruby documentation does that.

File: README.md [hunk 3]
```diff
@@ -2,9 +2,108 @@
...
+
+Deploy should only be called after signing. Revolution will throw an exception if it finds unsigned packages at this stage.
+```
+$ bundle exec revolution deploy --recipe-root recipes --config config.yml
+```
+
+Your config file should have the following contents:
+
+```yaml
+---
```
Comment (joshuajlai) [inline README.md]: Same here

**Pull request merged**